jenkins/ci.suse.de/pipelines/openstack-ardana-gerrit.Jenkinsfile

```diff
@@ -125,6 +110,12 @@ The following links can also be used to track the results:
       script{
         env.BUILD_RESULT = currentBuild.currentResult
         sh('''
+          automation-git/scripts/jenkins/jenkins-job-pipeline-report.py \
+            -j $JOB_NAME -b $BUILD_NUMBER --recursive \
+            --filter 'Declarative: Post Actions' \
+            --filter 'Setup workspace' \
+            pipeline-report.txt || :
```

Comment:
Totally a nit, but I would have had the jenkins-job-pipeline-report.py output to stdout and then use shell to redirect that to a file. It would just make any path handling clearer.